glambie/data/data_catalogue.py

```diff
@@ -38,7 +91,7 @@ def from_json_file(metadata_file_path: str) -> DataCatalogue:
         Returns
         -------
         DataCatalogue
-            data catalogue containing the metadata of datasets, the actual timeseries data will lazy loaded
+            data catalogue containing the metadata of datasets, the actual timeseries data will be lazily loaded
```

Comment:
The scentence is not clear because DataCatalogue isn't really a _true_ data catalogue; it actually contains all of the data once the individual timeseries are loaded. As ever, the implementation in the code warps the meaning of the words slightly away from their intuitive meaning.

I've hopefully made things a bit clearer now.